src/spec-node/imageMetadata.ts

```diff
@@ -494,8 +494,11 @@ export function getContainerEnvMetadata(containerEnv: Record<string, string> | u
 	if (!containerEnv) {
 		return '';
 	}
-
 	const keys = Object.keys(containerEnv);
-	const concatenatedEnv = keys.map(k => `ENV ${k}=${containerEnv![k]}`).join('\n');
+	// https://docs.docker.com/engine/reference/builder/#envs
+	const concatenatedEnv = keys.map(k => `ENV ${k}="${containerEnv[k]
+		.replace(/(?=["])/g, '\\') // escape double quotes
```

Comment:
Would backslashes also require escaping?

Reply:
Yes, I think we need to escape backslashes as well. Good call.

Reply:
Actually I think the original regex was probably correct: `replace(/(?=["\\$])/g, '\\')`. Looks like we need to escape `$` as well.